packages/core/src/operator/operatorRuntime.ts

```diff
@@ -493,9 +495,9 @@ async function listenForChallengesCallback(challengeNumber: bigint, challenge: C
         }
     } catch (error: any) {
         cachedLogger(`Error processing new challenge in listener callback: - ${error && error.message ? error.message : error}`);
+        // Propagate the error up by throwing it
+        throw error;
```

Comment:
Why is this throwing an error ?

Reply:
This has been removed, I mistakenly misunderstood it was needed to propogate errors up the stack.